examples/permissioned-voting-application/scripts/voting.js

```diff
@@ -0,0 +1,77 @@
+const { executeTransaction, TransactionType, SignType } = require("algob");
+
+/**
+ * Description: Converts Integer into Bytes Array
+ */
+function getInt64Bytes(x) {
```

Comment:
could you explain more this function? What is it doing?

Reply:
It is used to convert javascript integer into byte array.
Reference link: https://stackoverflow.com/a/56061448